docs/specifiers.md

```diff
@@ -8,9 +8,9 @@ You can specify versions of a package using the [Semantic Versioning scheme](htt
 
 For example, to install requests you can use:
 
-    $ pipenv install requests~=1.2
+    $ pipenv install requests~=1.1
 
-Pipenv will install version `1.2` and any minor update, but not `2.0`.
+Pipenv will install version `1.2` as it is a minor update, but not `2.0`.
```

Comment:
I wonder if we should also provide an example of using tilde to allow updating the major version, example ~1 would allow upgrading to I guess, anything?   It seems like a strange use case now that I mention it.

Reply:
I have added an example that allows micro version updates. Which would be more common usecase. 
I dont think we need an example for updating major version.